### PR TITLE
Applied UML Standards & simplified diagrams

### PR DESCRIPTION
**Pull request by ggrg**
At some point of reviewing https://github.com/mojaloop/documentation/pull/3 I decided that it will be easier to submit this PR, than to continue on commenting and requesting changes on files which you haven't edited, but just changed paths.

### CODE REVIEW

File: mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-participant-7.1.0.plantuml [hunk 1]
```diff
@@ -69,165 +70,101 @@ end box
 ' START OF FLOW
 
 group Get Participant's FSP Details
-    PAYER_FSP --> ALS_API: Request to get participant's FSP details \nGET - /participants/{TYPE}/{ID}?currency={CURRENCY}.\n<color #388724><b>Response code:</b> 202</color> \n<color #FF0000><b>Error code:</b> 200x, 300x, 310x, 320x</color>
+    PAYER_FSP ->> ALS_API: Request to get participant's FSP details \nGET - /participants/{TYPE}/{ID}?currency={CURRENCY}\n<color #388724><b>Response code:</b> 202</color> \n<color #red><b>Error code:</b> 200x, 300x, 310x, 320x</color>
```
Comment (mdebarros) [inline mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-participant-7.1.0.plantuml:73]: We use '-->' to denote an async callback response in our other sequence diagrams do we not?

File: mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-participant-7.1.0.plantuml [hunk 2]
```diff
@@ -69,165 +70,101 @@ end box
...
-
-        activate ALS_API
-        ALS_API --> PAYER_FSP: Callback: PUT - /participants/{TYPE}/{ID}?currency={CURRENCY}
-        deactivate ALS_API
-
-    else Empty list of End-Points returned or Error occurred
-
-        ALS_PARTICIPANT_HANDLER -> ALS_CENTRALSERVICE_ENDPOINT_CONFIG_DAO: Retrieve the Participant Callback End-point. \n<color #FF0000><b>Error code:</b> 200x, 310x, 320x</color>
+        ALS_PARTICIPANT_HANDLER --> ALS_API: Return list of Participant information
+        ALS_API ->> PAYER_FSP: Callback: PUT - /participants/{TYPE}/{ID}?currency={CURRENCY}
```
Comment (mdebarros) [inline mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-participant-7.1.0.plantuml:146]: We use '-->' to denote an async callback response in our other sequence diagrams do we not?

File: mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-participant-7.1.0.plantuml [hunk 3]
```diff
@@ -69,165 +70,101 @@ end box
...
+        ALS_PARTICIPANT_HANDLER -> ALS_PARTICIPANT_HANDLER: Match Participant Callback Endpoints for\nFSPIOP_CALLBACK_URL_PARTICIPANT_PUT_ERROR
 
-        ALS_PARTICIPANT_HANDLER -> ALS_API: Return \n<color #FF0000><b>Error code:</b> 200x, 310x, 320x</color>
-        activate ALS_API
-        ALS_API --> PAYER_FSP: Callback: PUT - /participants/{TYPE}/{ID}/error
-        deactivate ALS_API
-    else Empty list of switchEndpoint results returned
-        ALS_PARTICIPANT_HANDLER -> ALS_PARTICIPANT_HANDLER: Handle error\n<color #FF0000><b>Error code:</b> 200x</color>
+        ALS_PARTICIPANT_HANDLER --> ALS_API: Handle error\n<color #red><b>Error code:</b> 200x, 310x, 320x</color>
+        ALS_API ->> PAYER_FSP: Callback: PUT - /participants/{TYPE}/{ID}/error
```
Comment (mdebarros) [inline mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-participant-7.1.0.plantuml:160]: We use '-->' to denote an async callback response in our other sequence diagrams do we not?

File: mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-parties-7.2.0.plantuml [hunk 4]
```diff
@@ -74,149 +75,133 @@ end box
 ' START OF FLOW
 
 group Get Party Details
-    PAYER_FSP --> ALS_API: Request to get parties's FSP details \nGET - /parties/{TYPE}/{ID}?currency={CURRENCY}. \n<color #388724><b>Response code:</b> 202</color> \n<color #FF0000><b>Error code:</b> 200zx, 300x, 310x, 320x</color>
+    PAYER_FSP ->> ALS_API: Request to get parties's FSP details\nGET - /parties/{TYPE}/{ID}?currency={CURRENCY}\n<color #388724><b>Response code:</b> 202</color>\n<color #red><b>Error code:</b> 200zx, 300x, 310x, 320x</color>
```
Comment (mdebarros) [inline mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-parties-7.2.0.plantuml:78]: We use '-->' to denote an async callback response in our other sequence diagrams do we not?

File: mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-parties-7.2.0.plantuml [hunk 5]
```diff
@@ -74,149 +75,133 @@ end box
...
         '********************* Request Oracle Participant Information - END ************************
         |||
         '********************* Request Parties Information - START ************************
 
-        ALS_PARTICIPANT_HANDLER -> ALS_PARTIES_FSP_DAO: Request Parties information from FSP.\n<color #FF0000><b>Error code:</b> 200x</color>
+        ALS_PARTICIPANT_HANDLER -> ALS_PARTIES_FSP_DAO: Request Parties information from FSP.\n<color #red><b>Error code:</b> 200x</color>
 
         activate ALS_PARTIES_FSP_DAO
-        ALS_PARTIES_FSP_DAO --> PAYEE_FSP: Parties Callback to Destination: GET - /parties/{TYPE}/{ID}?currency={CURRENCY}. \n<color #388724><b>Response code:</b> 202</color> \n<color #FF0000><b>Error code:</b> 200x, 310x, 320x</color>
+        ALS_PARTIES_FSP_DAO ->> PAYEE_FSP: Parties Callback to Destination:\nGET - /parties/{TYPE}/{ID}?currency={CURRENCY}\n<color #388724><b>Response code:</b> 202</color>\n<color #red><b>Error code:</b> 200x, 310x, 320x</color>
```
Comment (mdebarros) [inline mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-parties-7.2.0.plantuml:133]: We use '-->' to denote an async callback response in our other sequence diagrams do we not?

File: mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-parties-7.2.0.plantuml [hunk 6]
```diff
@@ -74,149 +75,133 @@ end box
...
         deactivate ALS_PARTIES_FSP_DAO
         activate PAYEE_FSP
 
-        PAYEE_FSP --> ALS_API: Callback with Participant Information: PUT - /parties/{TYPE}/{ID}?currency={CURRENCY}. \n<color #FF0000><b>Error code:</b> 200x, 300x, 310x, 320x</color>
+        PAYEE_FSP ->> ALS_API: Callback with Participant Information:\nPUT - /parties/{TYPE}/{ID}?currency={CURRENCY}\n<color #red><b>Error code:</b> 200x, 300x, 310x, 320x</color>
```
Comment (mdebarros) [inline mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-parties-7.2.0.plantuml:137]: We use '-->' to denote an async callback response in our other sequence diagrams do we not?

File: mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-parties-7.2.0.plantuml [hunk 7]
```diff
@@ -74,149 +75,133 @@ end box
...
+        ALS_PARTICIPANT_HANDLER -> ALS_PARTICIPANT_HANDLER: Match PayerFSP Participant Callback Endpoints for\nFSPIOP_CALLBACK_URL_PARTIES_PUT
 
         '********************* Get PayerFSP Participant Information - END ************************
 
-        ALS_PARTICIPANT_HANDLER -> ALS_API: Send Participant Information to PayerFSP.
-        activate ALS_API
-        ALS_API --> PAYER_FSP: Callback with Parties Information: PUT - /parties/{TYPE}/{ID}?currency={CURRENCY}
-        deactivate ALS_API
+        ALS_PARTICIPANT_HANDLER --> ALS_API: Return Participant Information to PayerFSP
+        ALS_API ->> PAYER_FSP: Callback with Parties Information:\nPUT - /parties/{TYPE}/{ID}?currency={CURRENCY}
```
Comment (mdebarros) [inline mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-parties-7.2.0.plantuml:161]: We use '-->' to denote an async callback response in our other sequence diagrams do we not?

File: mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-parties-7.2.0.plantuml [hunk 8]
```diff
@@ -74,149 +75,133 @@ end box
...
+        ALS_PARTICIPANT_HANDLER -> ALS_PARTICIPANT_HANDLER: Match PayerFSP Participant Callback Endpoints for\nFSPIOP_CALLBACK_URL_PARTIES_PUT_ERROR
 
         '********************* Get PayerFSP Participant Information - END ************************
 
-        ALS_PARTICIPANT_HANDLER -> ALS_API: Return \n<color #FF0000><b>Error code:</b> 200x, 310x, 320x</color>
-        activate ALS_API
-        ALS_API --> PAYER_FSP: Callback: PUT - /participants/{TYPE}/{ID}/error
-        deactivate ALS_API
+        ALS_PARTICIPANT_HANDLER --> ALS_API: Handle error\n<color #red><b>Error code:</b> 200x, 310x, 320x</color>
+        ALS_API ->> PAYER_FSP: Callback: PUT - /participants/{TYPE}/{ID}/error
```
Comment (mdebarros) [inline mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-parties-7.2.0.plantuml:181]: We use '-->' to denote an async callback response in our other sequence diagrams do we not?

File: mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-parties-7.2.0.plantuml [hunk 9]
```diff
@@ -74,149 +75,133 @@ end box
...
+        ALS_PARTICIPANT_HANDLER -> ALS_PARTICIPANT_HANDLER: Match PayeeFSP Participant Callback Endpoints for\nFSPIOP_CALLBACK_URL_PARTIES_PUT_ERROR
 
         '********************* Get PayerFSP Participant Information - END ************************
 
-        ALS_PARTICIPANT_HANDLER -> ALS_API: Return \n<color #FF0000><b>Error code:</b> 200x, 310x, 320x</color>
-        activate ALS_API
-        ALS_API --> PAYEE_FSP: Callback: PUT - /participants/{TYPE}/{ID}/error
-        deactivate ALS_API
+        ALS_PARTICIPANT_HANDLER --> ALS_API: Handle error\n<color #red><b>Error code:</b> 200x, 310x, 320x</color>
+        ALS_API ->> PAYEE_FSP: Callback: PUT - /participants/{TYPE}/{ID}/error
```
Comment (mdebarros) [inline mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-parties-7.2.0.plantuml:200]: We use '-->' to denote an async callback response in our other sequence diagrams do we not?

File: mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-post-participants-7.2.1.plantuml [hunk 10]
```diff
@@ -96,22 +97,21 @@ group Post Participant's FSP Details
           "currency": "string"
         }
     end note
-    PAYER_FSP --> ALS_API: Request to add participant's FSP details POST - /participants. \n<color #388724><b>Response code:</b> 202</color> \n<color #FF0000><b>Error code:</b> 200x, 300x, 310x, 320x</color>
+    PAYER_FSP ->> ALS_API: Request to add participant's FSP details\nPOST - /participants\n<color #388724><b>Response code:</b> 202</color> \n<color #FF0000><b>Error code:</b> 200x, 300x, 310x, 320x</color>
```
Comment (mdebarros) [inline mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-post-participants-7.2.1.plantuml:100]: We use '-->' to denote an async callback response in our other sequence diagrams do we not?

File: mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-post-participants-7.2.1.plantuml [hunk 11]
```diff
@@ -146,87 +149,83 @@ group Post Participant's FSP Details
...
 
             '********************* Get PayerFSP Participant Information - END ************************
 
-            ALS_PARTICIPANT_HANDLER -> ALS_API: Return list of Participant information.
-
-            activate ALS_API
-            ALS_API --> PAYER_FSP: Callback: PUT - /participants/{requestId}
-            deactivate ALS_API
+            ALS_PARTICIPANT_HANDLER --> ALS_API: Return list of Participant information
+            ALS_API ->> PAYER_FSP: Callback: PUT - /participants/{requestId}
```
Comment (mdebarros) [inline mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-post-participants-7.2.1.plantuml:206]: We use '-->' to denote an async callback response in our other sequence diagrams do we not?

File: mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-post-participants-7.2.1.plantuml [hunk 12]
```diff
@@ -146,87 +149,83 @@ group Post Participant's FSP Details
...
+            ALS_PARTICIPANT_HANDLER -> ALS_PARTICIPANT_HANDLER: Match Participant Callback Endpoints for\nFSPIOP_CALLBACK_URL_PARTICIPANT_BATCH_PUT_ERROR
 
             '********************* Get PayerFSP Participant Information - END ************************
 
-            ALS_PARTICIPANT_HANDLER -> ALS_API: Return \n<color #FF0000><b>Error code:</b> 200x, 310x, 320x</color>
-            activate ALS_API
-            ALS_API --> PAYER_FSP: Callback: PUT - /participants/{requestId}/error
-            deactivate ALS_API
+            ALS_PARTICIPANT_HANDLER --> ALS_API: Handle error\n<color #FF0000><b>Error code:</b> 200x, 310x, 320x</color>
+            ALS_API ->> PAYER_FSP: Callback: PUT - /participants/{requestId}/error
```
Comment (mdebarros) [inline mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-post-participants-7.2.1.plantuml:226]: We use '-->' to denote an async callback response in our other sequence diagrams do we not?

**Review by mdebarros — CHANGES_REQUESTED** [review]
Thanks for this. Lots of visibility and consistency improvements. 

I do have a single issue for the callback line as per my comments.

**Comment by ggrg** [comment]
`We use '-->' to denote an async callback response in our other sequence diagrams do we not?`
Not really, we use '-->' to denote an async callback `Respond HTTP - 202 (Accepted)`.

**Review by mdebarros — APPROVED** [review]

**Pull request merged**